system/Commands/Utilities/Routes.php

```diff
@@ -109,4 +116,31 @@ public function run(array $params)
 
         CLI::table($tbody, $thead);
     }
+
+    private function getAutoRoutes(RouteCollection $collection): array
+    {
+        $defaultNamespace = $collection->getDefaultNamespace();
+        $finder           = new ControllerFinder($defaultNamespace);
+        $reader           = new ControllerMethodReader($defaultNamespace);
+
+        $tbody = [];
+
+        foreach ($finder->find() as $class) {
+            $output = $reader->read(
+                $class,
+                $collection->getDefaultController(),
+                $collection->getDefaultMethod()
```

Comment:
method calls should be moved to before loop when possible

Reply:
Yes, they are not changed. Thanks.

Moved.